packages/zk-certificates/contracts/VerificationSBT.sol

```diff
@@ -3,12 +3,24 @@ pragma solidity ^0.8.9;
 
 import "./interfaces/IVerificationSBT.sol";
 import "./interfaces/IVerifierWrapper.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 
 /// @author Galactica dev team
-/// @title A global smart contract that store verification SBT, minted by dApp for users submitting zk proofs
-contract VerificationSBT is IVerificationSBT {
+/// @title A global smart contract that store verification SBTs, minted by dApp for users submitting zk proofs
+contract VerificationSBT is IVerificationSBT{
```

Comment:
Shouldn't this contract also be a `IERC721`?

Reply:
In my understanding this contract is not a ERC721 if it is not implementing all those functions. I think the reverts can also be useful because we can give revert error message why this is not transferable.